CMSgov/mint-app
# [NOREF] add position-relative to the header buttons

# NOREF

## Changes and Description

- Addressing issue in which the header carets are not scrolling along with the headers (https://oddball.slack.com/archives/C059QQA3HK5/p1699375684730169?thread_ts=1699369479.745089&cid=C059QQA3HK5)
- adding `position-relative` to the parent `button` tag of the absolutely positioned svg



## How to test this change
1. reduce browser size to a smaller than tablet screen size and locate any tables.
2. ensure that the carets are now scrolling along with the copy



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
